### ISSUE TITLE
Calculate total power

### ISSUE THREAD

**Issue by jackonelli** [BryggansBryggeri/electrical#38]
We have ~11kW total power for 16A 3-phase
With heater at 10kW, 1kW remains.

Pump ~550W according to specs (not accounted for losses in VFD?)
Fan ~62W
Lamps a few watts each
Not a lot of wiggle room!

UPDATE: Adjusted figures

**Comment by JonathanFager** [comment]
Lamps are 5W each

**Comment by JonathanFager** [comment]
Worst case: Fuses trip. Nothing breaks right? @swebro

**Comment by JonathanFager** [comment]
Fan is 0.062kW

**Comment by k-forss** [comment]
If we were to draw to much, what could we do about it? Not too much other than limiting the SSR duty cycles. So I think that it is better to measure the currents, for now we could try to borrow a power meter and if we are close we could add one in the system

**Comment by JonathanFager** [comment]
Exactly. Our actors could have a property that is powerDraw and Bryggio could make sure that powerDraw never exceeds what we can supply.

For sure Plejd have nice power meeters just laying around.
